.eslintrc.json

```diff
@@ -2,5 +2,13 @@
    "env": {
       "node": true
    },
-   "extends": "@silvermine/eslint-config/node"
+   "extends": "@silvermine/eslint-config/node",
+
+   "rules": {
+      "prefer-template": "off",
+      "padding-line-between-statements": [
+         "error",
+         { "blankLine": "any", "prev": [ "*" ], "next": "return" }
```

Comment:
This is actually turning it off. I should have marked this `"BlankLine": "any"` means "leave a blank line or not". 

For clarity, I added this and `prefer-template` because of the weird loop we get into with the plugin and config. We're not pushing the config version bump until this is there, but I need to update the config now to make linting on the new indent rule work.

Reply:
Ok, thanks. I misread it. I see it now